RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.cc

```diff
@@ -704,10 +837,35 @@ template <typename TILES>
 void PatternRecognitionbyCLUE3D<TILES>::fillPSetDescription(edm::ParameterSetDescription &iDesc) {
   iDesc.add<int>("algo_verbosity", 0);
   iDesc.add<double>("criticalDensity", 4)->setComment("in GeV");
-  iDesc.add<int>("densitySiblingLayers", 3);
+  iDesc.add<double>("criticalSelfDensity", 0.15 /* roughly 1/(densitySiblingLayers+1) */)
```

Comment:
The default value does not correspond to what reported in the comment: please fix either one

Reply:
> The default value does not correspond to what reported in the comment: please fix either one

`roughly` says it all, otherwise, I could have hard-coded the value in `C++`.